apps/ui/src/components/Ui/Composer.vue

```diff
@@ -1,149 +1,83 @@
 });
-
-const inputValueLength = computed(() => inputValue.value.length);
-
-const showError = computed<boolean>(
-  () =>
-    !!props.error &&
-    (isDirty.value || props.definition.default !== inputValue.value)
-);
 </script>
 
 <template>
-  <div
-    ref="editorContainerRef"
-    class="s-base"
-    :class="{
-      's-error': showError
-    }"
-  >
-    <div class="s-label s-toolbar">
-      <label class="text-sm hidden s-label-char-count whitespace-nowrap">
-        <template v-if="inputValueLength >= 0 && definition.maxLength">
-          {{ _n(inputValueLength) }} / {{ _n(definition.maxLength) }}
-        </template>
-      </label>
-      <div class="grow"></div>
-      <UiTooltip title="Add heading text">
-        <button
-          type="button"
-          class="p-1 size-[26px] leading-[18px] hover:text-skin-link rounded focus-visible:ring-1"
-          @click="editor.heading"
-        >
-          H
-        </button>
-      </UiTooltip>
-      <UiTooltip title="Add bold text">
-        <button
-          type="button"
-          class="p-1 size-[26px] leading-[18px] font-bold hover:text-skin-link rounded focus-visible:ring-1"
-          @click="editor.bold"
-        >
-          B
-        </button>
-      </UiTooltip>
-      <UiTooltip title="Add italic text">
-        <button
-          type="button"
-          class="p-1 size-[26px] leading-[18px] italic hover:text-skin-link rounded focus-visible:ring-1"
-          @click="editor.italic"
-        >
-          <span class="font-display !text-[17px] !font-normal">I</span>
-        </button>
-      </UiTooltip>
-      <UiTooltip title="Add a link" class="size-[26px]">
-        <button
-          type="button"
-          class="p-1 size-[26px] leading-[18px] italic hover:text-skin-link rounded focus-visible:ring-1"
-          @click="editor.link"
-        >
-          <IS-link class="size-[18px]" />
-        </button>
-      </UiTooltip>
-      <UiTooltip title="Add an image" class="size-[26px]">
-        <label
-          class="flex justify-center p-1 size-[26px] leading-[18px] italic hover:text-skin-link rounded focus-visible:ring-1"
-        >
-          <input
-            ref="editorFileInputRef"
-            type="file"
-            accept="image/*"
-            class="hidden"
-            :disabled="editor.uploading.value"
+  <div>
+    <div class="h-[40px] flex space-x-3">
+      <template v-if="editorType === 'markdown'">
+        <button type="button" @click="previewEnabled = false">
+          <UiLabel
+            :is-active="!previewEnabled"
+            text="Write"
+            class="border-transparent"
           />
-          <UiLoading
-            v-if="editor.uploading.value"
-            :size="14"
-            class="inline-block"
+        </button>
+        <button type="button" @click="previewEnabled = true">
+          <UiLabel
+            :is-active="previewEnabled"
+            text="Preview"
+            class="border-transparent"
           />
```

Comment:
Do we need a "Preview" tab when user can already preview same under visual editor

<img width="721" height="88" alt="Image" src="https://github.com/user-attachments/assets/e97e6baa-b8fd-4d22-b69b-3fc309192824" />